service/project.go

```diff
@@ -312,6 +312,7 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		PerfEnabled             bool                                `json:"perf_enabled"`
 		BuildBaronSettings      restModel.APIBuildBaronSettings     `json:"build_baron_settings"`
 		TaskAnnotationSettings  restModel.APITaskAnnotationSettings `json:"task_annotation_settings"`
+		ContainerSizes          map[string]model.ContainerResources `json:"container_sizes"`
```

Comment:
Can this use the rest model instead of the DB model? That way, it could use `ToService` to create the DB model.